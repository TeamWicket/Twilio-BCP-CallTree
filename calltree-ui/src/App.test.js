import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe("App", () => {
  it("should match snapshot of App Component", () => {
    expect(shallow(<App />)).toMatchSnapshot();
  })
})
