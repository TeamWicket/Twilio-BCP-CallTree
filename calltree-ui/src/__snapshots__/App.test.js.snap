// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot of App Component 1`] = `ShallowWrapper {}`;
